hackathon_site/hardware/permissions.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework.permissions import DjangoModelPermissions
+
+
+class UserHasHardareOrderViewPermission(DjangoModelPermissions):
+    def __init__(self):
```

Comment:
Not quite sure how this works, I followed a medium tutorial: https://medium.com/innoventes/django-protect-apis-using-djangomodelpermissions-cb1bf65c5b58